workflow/scripts/split-call-tables.py

```diff
@@ -124,6 +125,46 @@ def cleanup_dataframe(df):
     return df
 
 
+def modify_observations(value, modifier):
+    regex = re.compile("([0-9]+)(E|B|P|S|V|e|b|p|s|v)")
+    observations = ""
+    m = re.findall(regex, value)
+    for count, score in m:
+        if score != "E":
+            observations += count + modifier + score
+        else:
+            observations += count + score
+    return observations
+
+
+def modify_observations(value, modifier):
+    regex = re.compile("([0-9]+)(E|B|P|S|V|e|b|p|s|v)")
+    observations = ""
+    m = re.findall(regex, value)
+    for count, score in m:
+        if score != "E":
+            observations += count + modifier + score
+        else:
+            observations += count + score
+    return observations
+
+
+def join_short_obs(df, samples):
+    for sample in samples:
+        df[f"{sample}: short observations"] = ""
+        for allele, abbrev in [("ref", "R"), ("alt", "A")]:
+            df[f"{sample}: short observations"] += df[
+                f"{sample}: short {allele} observations"
+            ].apply(modify_observations, args=(abbrev))
```

Comment:
I think I would prefer to keep the observations unmodified, and just concatenate them with a comma or so. The reason: we want to migrate the JS parsing into an npm library, and there it has to deal with the original format for SROBS and SAOBS.

Reply:
I changed it the way you suggested. :)